frontend/src/store/csrf.js

```diff
@@ -27,15 +21,10 @@ async function csrfFetch(url, options = {}) {
     options.headers["Content-Type"] =
       options.headers["Content-Type"] || "application/json";
   }
-  options.headers["X-CSRF-Token"] = sessionStorage.getItem("X-CSRF-Token");
-  // call fetch with the url and the updated options hash
+
   const res = await fetch(url, options);
-  // if the response status code is 400 or above, then throw an error with the
-  // error being the response
   if (res.status >= 400) throw res;
-  // if the response status code is under 400, then return the response to the
-  // next promise chain
-  return res;
+  else return res;
```

Comment:
You don't need this else and it's cleaner without it
```suggestion
  return res;
```